timely/src/progress/frontier.rs

```diff
@@ -688,7 +690,7 @@ impl<'a, T: 'a> AntichainRef<'a, T> {
     ///```
     pub fn to_owned(&self) -> Antichain<T> where T: Clone {
         Antichain {
-            elements: self.frontier.to_vec()
+            elements: self.frontier.iter().cloned().collect()
```

Comment:
So you use `iter().cloned()` if you don't want to / can't consume an owned object and `into_iter()` if you can?